lms/templates/courseware/courseware-chromeless.html

```diff
@@ -121,6 +121,7 @@
     // it will report the height of its contents to the parent window when the
     // document loads, window resizes, or DOM mutates.
     if (window !== window.parent) {
+      document.body.className += ' view-in-mfe';
```

Comment:
Good idea, there's another place around removing padding around unit blocks where this will be helpful.